Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "non-empty-string"
-version = "0.2.4"
+version = "0.3.0"
```

Comment:
Since this PR is adding a new function, and updating some syntactic sugar, I think it's fine just to bump it to `0.2.5`